packages/example-app-base/src/utilities/SiteDefinition.types.ts

```diff
@@ -37,6 +63,11 @@ export interface ISiteDefinition<TPlatforms extends string = string> {
    * that need to show that message bar. You can define exclusions too.
    */
   messageBars?: ISiteMessageBarConfig[];
+
+  /**
+   * Defines the necessary information to populate the version switcher.
+   */
+  versionSwitcherDefinition?: IVersionSwitcherDefinition;
```

Comment:
Not a huge deal, but how much work would it be to require this prop always and provide a noop implementation? (i.e. a "null object"). Would save a bunch of null-checks throughout the site.

Reply:
Good point, I'm not actually sure why this was added in master with the properties optional. But at this point I'd prefer to just leave it (at least until another round of changes) in the interest of having it mirror master.